test/integration/lbp.js

```diff
@@ -0,0 +1,49 @@
+const {expect} = require('chai');
+const { ethers } = require("hardhat");
+const { parseEther } = ethers.utils;
+
+const init = require("../test-init.js");
+const {constants} = require('@openzeppelin/test-helpers');
+const expectEvent = require('@openzeppelin/test-helpers/src/expectEvent');
+
+const deploy = async () => {
+	const setup = await init.initialize(await ethers.getSigners());
+
+	setup.vault = await init.deployVault(setup);
+
+	setup.lbpFactory = await init.deployLBPFactory(setup);
+
+	setup.Lbp = init.Lbp(setup);
+
+	setup.token = await init.tokens(setup);
+
+	return setup
+}
+
+describe("LbpFactory", async () => {
+	let vault;
+	let setup;
+	let poolTokens;
+	let lbp;
+
+	context("test factory", async () => {
+		before('setup', async () => {
+			setup = await deploy();
+		});
+		it('deploys new LBP pool', async () => {
+			const receipt = await (await setup.lbpFactory.create(
+				"Test",
```

Comment:
pls don't use magic values and store all the parameters in a separate variable

Reply:
Changed all the magic values